aspnetcore/mvc/models/model-binding.md

```diff
@@ -23,6 +23,12 @@ Controllers and Razor pages work with data that comes from HTTP requests. For ex
 * Converts string data to .NET types.
 * Updates properties of complex types.
 
+### Simple and complex types
+
+
+A simple type is one of the predefined `struct` types such as `int`, `char`, `float`, `double`, and `bool`. For a complete list, see [Simple types](/dotnet/csharp/language-reference/language-specification/types#835-simple-types). A complex type is a type that is not a simple type, such as a `class` or  [`record`](/dotnet/csharp/language-reference/builtin-types/record).
```

Comment:
In MVC, simple types are basically defined as a type that contains a `TypeConverter` from string or TryParse method, that is a very different definition from the language

Reply:
In this paragraph there is a briefly explanation https://learn.microsoft.com/en-us/aspnet/core/mvc/advanced/custom-model-binding?view=aspnetcore-6.0#model-binding-review

Reply:
https://github.com/dotnet/AspNetCore.Docs/issues/27389